pyiron_base/database/performance.py

```diff
@@ -0,0 +1,170 @@
+import pandas as pd
```

Comment:
Can you add the copyright statement on top?

Reply:
Done!